_topic_map.yml

```diff
@@ -1938,27 +1938,29 @@ Topics:
   - Name: Recovering from expired control plane certificates
     File: scenario-3-expired-certs
 ---
+Name: Migrating from OpenShift Container Platform 3 to 4
```

Comment:
Just checking whether this new top-level folder has already been run by/approved by @vikram-redhat and the content strategists?

Reply:
Absolutely. I would not have started such a big move without approval from the start. Vikram, Audrey Spaulding, and Michelle Bearer are behind this. :-)